### PR TITLE
cleanup 

### LINKED ISSUE #3: Decide on future of "run.sh"
Should just document the use of Ansible and avoid a wrapper .sh that could drive the wrong behavior.

### LINKED ISSUE #4: Rename top-level "template" directory to "openshift-templates"
To avoid confusions around the generic word "template", I'd like to vote for renaming the top level directory from `template` to `openshift-templates`

### LINKED ISSUE #13: Clean Up Params Files
`build_params` vs `deploy_params` is not being followed by convention, and it's confusing for new users. Let's consolidate these into something that is more meaningful. Perhaps each app has its own folder including all build / dev / test / uat params. I'm not a fan of 1 file per directory

### LINKED ISSUE #37: Move to Dev/Demo project structure from dev/test/uat
We aren't using the extra projects in practice and they are just consuming extra resources and adding maintenance overhead. Where people need extra projects, obviously they can customize

### PR DESCRIPTION
**Pull request by sherl0cks**
resolves #3 
resolves #4 
resolves #13 
resolves #37 

Ready to go

### CODE REVIEW

**Comment by sherl0cks** [comment]
yikes - we have some regressions. hold on.

File: openshift-templates/jenkins-slave-pod/template-with-secret.json [hunk 1]
```diff
@@ -37,6 +37,17 @@
...
+            "metadata": {
+                "labels": {
+                    "build": "${NAME}",
+                    "role": "jenkins-slave"
+                },
+                "name": "${NAME}"
+            }
+        },
+        {
+            "apiVersion": "v1",
```
Comment (sherl0cks) [inline openshift-templates/jenkins-slave-pod/template-with-secret.json]: FYI this probably should have happened with #42

**Comment by sherl0cks** [comment]
Ok looks good now

**Comment by oybed** [comment]
@sherl0cks a lot of great stuff and I'm good with it. However, to aid in case someone is still relying on the "old stuff", I cut a pre-release tag `v0.1.0` that we can point people to in case they need anything from the older version(s). At this point I'm good to merge this as-is. I have one minor suggestion, but I'll submit a follow-up PR to take care of that one. 

https://github.com/rht-labs/labs-ci-cd/releases/tag/v0.1.0

**Pull request merged**